dmutils/logging.py

```diff
@@ -61,16 +60,12 @@ def get_handlers(app):
     standard_formatter = CustomLogFormatter(LOG_FORMAT, TIME_FORMAT)
     json_formatter = JSONFormatter(LOG_FORMAT, TIME_FORMAT)
 
-    # Log to files if the path is set, otherwise log to stderr
-    if app.config['DM_LOG_PATH']:
-        handler = logging.FileHandler(app.config['DM_LOG_PATH'])
+    if app.config['DM_ENVIRONMENT'] in ['development', 'test']:
```

Comment:
I wonder if having a separate config variable in the apps for this would be more straightforward. Eg `DM_LOG_TYPE = "json"`. We can default to JSON here and then override it in development and test configs in the apps.